scripts/dash-interactive-graphs.py

```diff
@@ -0,0 +1,194 @@
+from dash import Dash, html, dcc, Input, Output
```

Comment:
We want a docstring here telling potential users and maintainers what this script does, with inputs and outputs. In a few words.

Reply:
file deleted